src/saltext/vmware/modules/vmc_sddc_clusters.py

```diff
@@ -0,0 +1,136 @@
+"""
+Salt execution module for Cluster management
+Provides methods to Retrieve, Create and Delete cluster in the target SDDC
+"""
+import logging
+
+from saltext.vmware.modules import vmc_sddc
+from saltext.vmware.utils import vmc_constants
+from saltext.vmware.utils import vmc_request
+from saltext.vmware.utils import vmc_templates
+
+log = logging.getLogger(__name__)
+
+__virtualname__ = "vmc_sddc_clusters"
+
+
+def __virtual__():
+    return __virtualname__
+
+
+def get(hostname, refresh_key, authorization_host, org_id, sddc_id, verify_ssl=True, cert=None):
```

Comment:
**suggestion** should be `list_` instead

Reply:
https://github.com/saltstack/salt-ext-modules-vmware/pull/221#discussion_r845685289